src/bin/tikv-server.rs

```diff
@@ -71,8 +71,8 @@ fn get_string_value<F>(short: &str,
 fn initial_log(matches: &Matches, config: &toml::Value) {
     let level = get_string_value("L",
                                  "server.log-level",
-                                 &matches,
-                                 &config,
+                                 matches,
```

Comment:
Btw, most of time, we won't meet these errors. After we update the rust and clippy, we must guarantee passing compile first.